tests/test_statbank.py

```diff
@@ -536,6 +537,17 @@ def test_transfer_no_auth_residuals(transfer_success: StatbankTransfer):
     assert len(search__dict__(transfer_success, fake_auth(), keep={})) == 0
 
 
+def test_transfer_has_auth_residuals(transfer_success: StatbankTransfer):
+    """Checks that auth information is actually detected."""
+    response = fake_post_response_transfer_successful()
+    response.cookies = requests.cookies.cookiejar_from_dict({"password": fake_pass()})
+    transfer_success.response = response
+
+    # Make sure none of these are in the object for security
```

Comment:
Change this comment to "make sure we now find the inserted auth, verifying that search__dict__ is doing its job". Or similar. Thanks!